test/Ocelot.UnitTests/Configuration/VersionPolicyCreatorTests.cs

```diff
@@ -0,0 +1,79 @@
+﻿using Ocelot.Configuration.Creator;
+using System.Net.Http;
+using Shouldly;
+using TestStack.BDDfy;
+using Xunit;
+
+namespace Ocelot.UnitTests.Configuration
+{
+    public class VersionPolicyCreatorTests
+    {
+        private readonly VersionPolicyCreator _creator;
+        private string _input;
+        private HttpVersionPolicy _result;
+
+        public VersionPolicyCreatorTests()
+        {
+            _creator = new VersionPolicyCreator();
+        }
+
+        [Theory]
+        [InlineData(VersionPolicies.RequestVersionOrLower)]
+        [InlineData(VersionPolicies.RequestVersionExact)]
+        [InlineData(VersionPolicies.RequestVersionOrHigher)]
+        public void should_create_version_policy_based_on_input(string versionPolicy)
+        {
+            this.Given(_ => GivenTheInput(versionPolicy))
+                .When(_ => WhenICreate())
+                .Then(_ => ThenTheResultIs(versionPolicy))
+                .BDDfy();
```

Comment:
Don't use BDDfy for unit tests! It is useless!
This is old style, and BDDfy will be removed from unit testing project soon.

Use [AAA pattern](https://www.bing.com/search?q=AAA+pattern+unit+test) ❗